pkg/apis/core/v1alpha1/types_common.go

```diff
@@ -34,6 +34,9 @@ const (
 type LastError struct {
 	// A human readable message indicating details about the last error.
 	Description string `json:"description"`
+	// Name of the task which caused this last error
+	// +optional
+	TaskID string `json:"taskName,omitempty"`
```

Comment:
```suggestion
	TaskID *string `json:"taskName,omitempty"`
```

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
done